### DISCUSSION TITLE
Bug in Mac is USE_DOUBLE_PRECISION is set to OFF

### DISCUSSION THREAD

**Discussion by dwilches** [bulletphysics/bullet3#3478] [General]
Hello,

I ran into an issue when compiling Bullet in Mac OS 10.10.2 using the default CMake values.
The problem was that some demos, for example the RagDollDemo, were acting funny: in that demo the rag dolls must appear and fall to the ground, and remain there. But in my case, the rag dolls fell and then started sliding towards the borders of the screen, until they were gone.
I could confirm the same behaviour with my own programs, were things just slide when they must lay on ground.

The way I fixed it was by setting USE_DOUBLE_PRECISION to ON in CMake, before generating my Makefile.

I confirmed that this setting fixed the problem by turning it OFF and ON several times, and seeing the behaviour was consistent each time.

**Comment by aaronfranke** [comment]
Is this still an issue?
